m4_sessions/1810-inning/creativity.md

```diff
@@ -5,31 +5,45 @@ Dennis Miller, Laura Whitaker, Jesse McFadden, Paul Kim
 
 ## Name of Session: 
 
-Create and Engineer
+Creativity
 
 ## Session Description (Why should students come to this session): (EX: Are you interested in biological enhancement? Drop in to learn about biohacking.)
 
+How does it help with problem solving? How do you practice it? How does it affect technology and vice versa? Maybe find some answers or leave with more questions… Either way we hope to get you thinking.
 
 ## Session Outcomes 
 
 (What will students learn/leave with?): 
 
 1) Feeling inspired and energized
-2)Takes them out of their element 
-3) Having a next step.
-4)feeling like they made a connection
+2) Takes them out of their element 
+3) Having a next step
+4) Feeling like they made a connection
 
      Best practices:  No section should be longer than 20 minutes
      Create a session that YOU would want to attend
      Share resources in your slack channel
      All sessions NEED to have a closing activity, reflection, discussion
 ## Session Outline
 
-     Ice Breaker: What is one creative thing you did this week?
-     
-0) Ice Breaker: What is one creative thing you did this week? 1) Inspired/Energized  2) Have done something that made them uncomfortable(feeling good about it)  3) Feeling like they made a connection. (edited)
+10min - Dennis: Hand out pieces of colored paper / Ice breaker / Question - What is one creative thing you did this week?
+10min - Paul: Object exercise - What are other use cases for…. Ex. keycard, mug, pen, cellphone
+5min - Jesse: Intro / Set the tone
 
-1)Exquisite corpse. 2) Ted talk on creativity(or something) 3) Brainstorming/Discussion
+5min - break
+
+3min - Video (https://www.youtube.com/watch?v=LgRSjAi7SVM)
+5min -Discussion w/ prompted questions (break into 4 groups based on colors)
+3min - Video (https://www.youtube.com/watch?v=J3Om3vknLl0)
+5min -Discussion w/ prompted questions
+3min - Video (https://www.youtube.com/watch?v=zizkcQ1IXoA)
+5min -Discussion w/ prompted questions
```

Comment:
What are the prompted questions for each part? 

Write the questions on the board.